### DISCUSSION TITLE
UDP-RPL EXAMPLE

### DISCUSSION THREAD

**Discussion by wdargie** [contiki-ng/contiki-ng#2454] [Q&A] [answered]
In the rpl-udp example I wish to change the default routing config to RPL classic as opposed to rpl-lite, but I could not find the project configuration file. I changed contiki-default-conf.h but did not change the udp-server.c or udp-client.c configuration  A little tip would be appreciated.

**Comment by arurke** [comment]
You can select the RPL implementation via the Makefile. Details can be found in the docs. [here](https://docs.contiki-ng.org/en/develop/doc/programming/RPL.html#implementations).

**Comment by wdargie** [comment]
Thank you. Much appreciated.

<http://wwwpub.zih.tu-dresden.de/~dargie/>

________________________________
Von: Andreas Urke ***@***.***>
Gesendet: Sonntag, 4. Juni 2023 10:26:21
An: contiki-ng/contiki-ng
Cc: Dargie, Waltenegus; Author
Betreff: Re: [contiki-ng/contiki-ng] UDP-RPL EXAMPLE (Discussion #2454)


You can select the RPL implementation in the Makefile. Details can be found in the docs. here<https://docs.contiki-ng.org/en/develop/doc/programming/RPL.html#implementations>.

**Comment by arurke** [comment, marked as answer]
Happy to help. PS! If you mark my reply as the correct answer it will be
more useful for others with the same issue.
